config/spryk/spryks/Spryker/Glue/BackendApi/Mapper/AddGlueBackendApiResponseMapper.yml

```diff
@@ -0,0 +1,114 @@
+spryk: template
+description: "Adds a new Response mapper that maps {{ zedDomainEntity}} transfers to a GlueReponseTransfer."
+mode: both
+level: 2
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    resource:
+        description: "Defines the resource name this controller will work with."
+
+    resourceDataObject:
+        description: "Defines the Transfer that will be used in the Methods. Do not use Transfer as suffix, it will be added automatically. Example: Customer (ok) CustomerTransfer (wrong)"
+        isOptional: true # Is not needed for all methods
+        value: '{{ resource | singular | dashToCamelCase | ucfirst }}'
+
+    template:
+        value: Common/Method.php.twig
+
+    target:
+        value: "{{ organization }}\\Glue\\{{ module }}\\Mapper\\GlueResponse{{ resourceDataObject }}Mapper"
+
+preSpryks:
+    - AddGlueBackendApiController
+
+postSpryk:
+    - AddGlueBackendApiMapperMethod:
```

Comment:
Can you check if it's possible to use the `AddMethod` Spryk instead of the newly created one? If we can use that it would be better as we would have fewer Spryks.